java-maps-fleetengine/google-maps-fleetengine/pom.xml

```diff
@@ -1,116 +0,0 @@
-<?xml version='1.0' encoding='UTF-8'?>
-<project xmlns="http://maven.apache.org/POM/4.0.0" xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance" xsi:schemaLocation="http://maven.apache.org/POM/4.0.0 http://maven.apache.org/xsd/maven-4.0.0.xsd">
-  <modelVersion>4.0.0</modelVersion>
-  <groupId>com.google.maps</groupId>
-  <artifactId>google-maps-fleetengine</artifactId>
```

Comment:
This hasn't been published. Good.

![image](https://github.com/googleapis/google-cloud-java/assets/28604/59f31ce3-8b08-45dd-a249-97b6a0ff00f4)